packages/google_maps_flutter/google_maps_flutter/example/pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   google_maps_flutter:
     path: ../
   flutter_plugin_android_lifecycle: ^1.0.0
+  google_maps: ^3.0.0
```

Comment:
remove

Reply:
I will revert the `example`. I think `flutter create` was run inside the `example` dir, and it added a lot of stuff that is not needed for this PR.